### ISSUE TITLE
Unexpected exception of type BadZipFile occured: ('File is not a zip file',)

### ISSUE THREAD

**Issue by sachsgiri** [eliasgranderubio/dagda#31]
### Short description

I was trying to initialize vulndb with " vuln --init" cli command.

### Reproduction steps

1. Start Dagda server with the parameters which are default. server: port localhost:5000 and mongo port default.
2. Type the Dagda CLI command "python3 dagda.py vuln --init"
3. "python3 dagda.py vuln --init_status" after few minutes

### Expected results

I was expecting something like this:

{
    "status": "Initialized",
    "timestamp": "2018-06-06 05:45:03.871051"
}

### Actual results

{
    "status": "Unexpected exception of type BadZipFile occured: ('File is not a zip file',)",
    "timestamp": "2018-06-06 06:25:29.540764"
}

### Backtrace

I don't know where to find it.

**Comment by eliasgranderubio** [comment]
Hi @sachsgiri ,

I've just initialize my vulndb from an empty database and it works fine:
```sh
~$ python3 dagda.py vuln --init_status
{
    "status": "Initializing",
    "timestamp": "2018-06-08 17:09:16.655259"
}

~$ python3 dagda.py vuln --init_status
{
    "status": "Updated",
    "timestamp": "2018-06-08 17:24:31.443086"
}
```

Probably, you have had a network error with NIST database because it is the only site where Dagda downloads information in ZIP extension. You can check this in the [DBComposer](https://github.com/eliasgranderubio/dagda/blob/master/dagda/vulnDB/db_composer.py) python file.

Anyway, you must connect to your mongodb database and drop the next collections:
```sh
~$ mongo
> use vuln_database
switched to db vuln_database
> db.getCollectionNames()
[
	"bid",
	"bid_info",
	"cve",
	"cve_info",
	"exploit_db",
	"exploit_db_info",
	"image_history",
	"init_db_process_status",
	"rhba",
	"rhba_info",
	"rhsa",
	"rhsa_info"
]
> db.cve.drop()
true
> db.cve_info.drop()
true
> db.bid.drop()
true
> db.bid_info.drop()
true
> db.rhsa.drop()
true
> db.rhsa_info.drop()
true
> db.rhba.drop()
true
> db.rhba_info.drop()
true
> db.exploit_db.drop()
true
> db.exploit_db_info.drop()
true
```

After that, run again `vuln --init`. If problem persists, please let me know.

**Comment by eliasgranderubio** [comment]
This issue is very closed with #32 so I close this issue and I redirect you to the other one if your issue persists.
